search-sentiments-map/public/style.css

```diff
@@ -181,11 +193,23 @@ input:checked + .slider:before {
   overflow-y: auto;
 }
 
+#trends-title {
+  display: inline-block;
+}
+
+#switch-trends-click {
+  color: #2196F3;

```

Comment:
Generally for more obscure colors (hex codes) I think I learned from Haylee that we leave a comment to describe what it is, but maybe @hayleejane3 can comment more on this if this is necessary or not.

Reply:
Ok I tried to give names to these colors but I really just picked the one I like with an online color picker